Assets/HoloToolkit/SpatialSound/Scripts/IAudioInfluencer.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Interface that is implemented by any class that wishes to influence how an audio source sounds.
+    /// </summary>
+    public interface IAudioInfluencer
+    {
+        /// <summary>
+        /// Applies an audio effect.
+        /// </summary>
+        /// <param name="emitter">The GameObject on which the effect is to be applied.</param>
+        /// <param name="audioSource">The AudioSource that will be impacted by the effect.</param>
+        void ApplyEffect(GameObject emitter,
```

Comment:
Hmm given that the signatures are same I would have expected ApplyEffect to perhaps return the effect GO so I don't have to do a GetComponent again. And that effect passed into RemoveEffect. But I guess this works too.